build.gradle

```diff
@@ -67,13 +67,14 @@ dependencies {
 
 
     implementation "com.github.Oblarg.Oblog:lib:3.1.0"
-    implementation name: 'jetty-all-9.4.0.v20161208-uber'
+
     implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
 
     implementation 'org.ghrobotics.FalconLibrary:wpi:2020.2.2'
 
     implementation 'com.sparkjava:spark-core:2.8.0'
     implementation 'com.sparkjava:spark-template-velocity:2.7.1'
+    implementation "com.revrobotics.frc:SparkMax-java:1.5.2"
```

Comment:
I don't think you need this line, but in any case, where are you using REV products?

Reply:
It was for the NEO in the intake